node/forest_libp2p/bitswap/src/internals/event_handlers.rs

```diff
@@ -72,25 +72,29 @@ fn handle_inbound_request<S: BitswapStoreRead>(
     store: &S,
     request: &BitswapRequest,
 ) -> Option<BitswapResponse> {
-    match request.ty {
-        RequestType::Have => {
-            metrics::message_counter_inbound_request_have().inc();
-            let have = store.contains(&request.cid).ok().unwrap_or_default();
-            if have || request.send_dont_have {
-                Some(BitswapResponse::Have(have))
-            } else {
-                None
+    if request.cancel {
```

Comment:
maybe we could just have a guard `if request.cancel { return None; }` to reduce the indentation of the rest of the method

Reply:
fixed